packages/react-adminer/src/hooks/useSetUrlFilter.ts

```diff
@@ -1,18 +1,9 @@
+import { deserializeParse } from '../utils/config';
 import useStateParams from './useStateParams';
 
 export default function useSetUrlFilter(): any {
-	const [filterConfig, setFilter] = useStateParams<any>(
-		undefined,
-		'f',
-		(v?: string) => {
-			try {
-				const parsed = JSON.parse(v ?? 'empty');
-				return parsed;
-			} catch {
-				return {};
-			}
-		},
-		(v: string) => JSON.stringify(v),
+	const [filterConfig, setFilter] = useStateParams<any>(undefined, 'f', deserializeParse, (v: string) =>
```

Comment:
tu druhou funkci na stringify bych dal taky do utils a jeste pred to dal prefix json